src/DonationForms/Controllers/DonationFormViewController.php

```diff
@@ -25,6 +25,7 @@ public function show(DonationFormViewRouteData $data): string
             $donationForm->settings
         );
 
+        ob_clean();
```

Comment:
Does calling `ob_clean()` actually do anything? We area already starting and closing the buffer in the render method after this using `ob_start()` and `ob_get_clean()`.

Reply:
Yeah, if you add `echo 'ARGH!'` on that line and remove the `ob_clean()` it will show up. Double-check me, though.

Reply:
Ah I see, how about we put the `ob_clean()` right before `ob_start()` then in the vewModel render methods to make sure it's always cleared before starting.

Reply:
I thought about that. The only thing that held me back was the idea that perhaps the `viewModel` shouldn't worry about clearing the previous buffer, as much as capturing its own output. Separations of concerns sort of thing. What do you think?

Reply:
hmm, yeah I guess that makes sense. carry on then lol